src/plugins/intel_cpu/tests/unit/graph/dummy_node.hpp

```diff
@@ -21,7 +21,7 @@ class DummyNode : public Node {
             const std::string& type,
             const GraphContext::CPtr context,
             LayoutType layout = LayoutType::ncsp,
-            Edge::LOOK in_place_direction = Edge::LOOK::LOOK_UP,
+            int in_place_direction = Edge::LOOK::LOOK_UP,
```

Comment:
Why do we need change the type here?

Reply:
I want to create a dummy node without inplace. The original use the 'Edge::LOOK in_place_direction' to setup the in/out inplace config. But there is no enum item in Edge::LOOK meaning not LOOK_UP and not LOOK_DOWN. So I change the type to int and use 0 to stand for not inplace in both direction.